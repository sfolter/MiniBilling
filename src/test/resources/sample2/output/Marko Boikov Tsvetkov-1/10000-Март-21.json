{
    "documentDate": "2022-08-10T11:10:11.625748300",
    "documentNumber": "10000",
    "consumer": "Marko Boikov Tsvetkov",
    "reference": "1",
    "totalAmount": 73.08,
    "lines": [
        {
            "index": 1,
            "quantity": 69.6,
            "lineStart": "2021-01-01T09:00:00Z",
            "lineEnd": "2021-02-28T23:59:59Z",
            "product": "gas",
            "price": 0.7,
            "priceList": 1,
            "amount": 48.72
        },
        {
            "index": 2,
            "quantity": 17.4,
            "lineStart": "2021-03-01T00:00:00Z",
            "lineEnd": "2021-03-17T10:20:00Z",
            "product": "gas",
            "price": 1.4,
            "priceList": 1,
            "amount": 24.36
        }
    ]
}