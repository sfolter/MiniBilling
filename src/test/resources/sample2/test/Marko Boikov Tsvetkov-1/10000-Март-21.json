{
    "documentDate": "2022-08-08T16:27:28.836359900",
    "documentNumber": "10000",
    "consumer": "Marko Boikov Tsvetkov",
    "reference": "1",
    "totalAmount": 243.6,
    "lines": [
        {
            "index": 2,
            "quantity": 87,
            "lineStart": "2021-01-01T09:00:00Z",
            "lineEnd": "2021-03-17T10:20:00Z",
            "product": "gas",
            "price": 1.4,
            "priceList": 1,
            "amount": 121.8
        },
        {
            "index": 2,
            "quantity": 87,
            "lineStart": "2021-01-01T09:00:00Z",
            "lineEnd": "2021-03-17T10:20:00Z",
            "product": "gas",
            "price": 1.4,
            "priceList": 1,
            "amount": 121.8
        }
    ]
}