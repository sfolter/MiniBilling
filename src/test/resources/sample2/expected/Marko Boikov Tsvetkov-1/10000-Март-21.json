{
  "documentNumber": "10000",
  "consumer": "Marko Boikov Tsvetkov",
  "reference": "1",
  "totalAmount": 74.73,
  "lines": [
    {
      "index": 1,
      "quantity": 67.251,
      "lineStart": "2021-01-01T09:00:00Z",
      "lineEnd": "2021-02-28T21:59:59Z",
      "product": "gas",
      "price": 0.7,
      "priceList": 1,
      "amount": 47.08
    },
    {
      "index": 2,
      "quantity": 19.749,
      "lineStart": "2021-02-28T22:00:00Z",
      "lineEnd": "2021-03-17T10:20:00Z",
      "product": "gas",
      "price": 1.4,
      "priceList": 1,
      "amount": 27.65
    }
  ]
}